xgboost_ray/main.py

```diff
@@ -475,13 +475,14 @@ def train(self, rabit_args: List[str], return_bst: bool,
                 # bytearray type gets lost in remote actor call
                 kwargs["xgb_model"] = bytearray(kwargs["xgb_model"])
 
-        if "nthread" not in local_params:
+        if "nthread" not in local_params and "n_jobs" not in local_params:
             if num_threads > 0:
                 local_params["num_threads"] = num_threads
             else:
                 local_params["nthread"] = sum(
                     num
                     for _, num in ray.worker.get_resource_ids().get("CPU", []))
+                local_params["n_jobs"] = local_params["nthread"]
```

Comment:
n_jobs and nthreads are aliases of each other. The xgboost documentation says that new users should use the former. xgboost-dask overwrites both with the same value, so that's what I've done here too.

Reply:
Thanks for the explanation!